galaxy_ng/app/viewsets.py

```diff
@@ -22,41 +22,48 @@
 # on the remote field and can't find a viewset name for galaxy's ContainerRegistryRemote model.
 
 
-class ContainerRegistryRemoteViewSet(pulp_viewsets.NamedModelViewSet, mixins.RetrieveModelMixin):
+# Added to keep the DRF forms from breaking
+class ContainerRegistryRemoteViewSet(
+    pulp_viewsets.NamedModelViewSet,
+    mixins.RetrieveModelMixin,
+):
     queryset = models.ContainerRegistryRemote.objects.all()
     serializer_class = serializers.ContainerRegistryRemoteSerializer
     permission_classes = [access_policy.ContainerRegistryRemoteAccessPolicy]
-    endpoint_name = "execution-environments-registry-detail"
+    endpoint_name = "galaxy/registry-remote"
```

Comment:
should the `endpoint_name` prefixes match on this module... either galaxy or galaxy_ng?